Sources/Core/Client/WebSocket/WebSocket.swift

```diff
@@ -73,6 +62,18 @@ public final class WebSocket {
         disconnect()
     }
     
+    /// Subscribe for websocket response.
+    /// - Parameter completion: a completion block (see `ClientCompletion`).
+    /// - Returns: a subscription.
+    public func response(_ completion: @escaping ClientCompletion<WebSocket.Response>) -> Subscription {
+        return rxResponse.bind(to: completion)
```

Comment:
`bindOnce` instead of `bind`?

Reply:
Nope. You are observing events continuously.